### DISCUSSION TITLE
Welcome to baresip-studio Discussions!

### DISCUSSION THREAD

**Discussion by juha-h** [juha-h/baresip-studio#280] [General]
Welcome to baresip-studio Discussions!

**Comment by bjlockie** [comment]
This is a great app.
A lot of other apps collect insane data.

**Comment by misha-franz** [comment]
Hi @juha-h , interested in porting your app into Android based deskphones of the major phones manufacturer?

**Comment by juha-h** [comment]
I would need to know more details in order to answer.  Please contact me via email.

**Comment by misha-franz** [comment]
Which email I can use to contact you?

**Comment by juha-h** [comment]
You will find it in file https://github.com/juha-h/baresip-studio/blob/master/app/src/main/res/values/strings.xml.
